src/Conversion/KrnlToLLVM/KrnlToLLVM.cpp

```diff
@@ -613,8 +614,12 @@ class KrnlEntryPointOpLowering : public OpRewritePattern<KrnlEntryPointOp> {
   // returned, otherwise return nullptr.
   Value callApi(PatternRewriter &rewriter, Location loc, ApiRegistry registry,
       API apiId, ArrayRef<Value> params) const {
+    SmallVector<Type, 1> outputTys;
```

Comment:
We should change the API spec declaration on L587 to change voidTy to empty list.

Reply:
Currently, if I change the voidTy in the API spec declaration to an empty list, it will cause `getFuntionTy` to crash, even if it still compiles.

https://github.com/onnx/onnx-mlir/blob/cc13f0f7dac2287cfd774e6dcc986ba91159222d/src/Conversion/KrnlToLLVM/KrnlToLLVM.cpp#L425-L428

Reply:
Currently, if I change the voidTy to an empty list in the API spec declaration, it will crash on `getFuntionTy`, even though it compiles. 

https://github.com/onnx/onnx-mlir/blob/cc13f0f7dac2287cfd774e6dcc986ba91159222d/src/Conversion/KrnlToLLVM/KrnlToLLVM.cpp#L425-L428